gradle/dependencies.gradle

```diff
@@ -35,6 +36,22 @@ def build = [
     gradleErrorPronePlugin  : "net.ltgt.gradle:gradle-errorprone-plugin:0.0.8",
     guava                   : "com.google.guava:guava:21.0",
     jsr305Annotations       : "com.google.code.findbugs:jsr305:3.0.2",
+
+    // android stuff
+    buildToolsVersion: "27.0.2",
+    compileSdkVersion: 27,
+    ci: "true" == System.getenv("CI"),
+    minSdkVersion: 16,
+    targetSdkVersion: 27,
+
+    gradlePlugins: [
+        android: "com.android.tools.build:gradle:2.3.1",
```

Comment:
You should use 3.0.1, which is the current stable release. AutoDispose is on an older one but only because I haven't updated it yet :)